### ISSUE TITLE
Elementos <volume>, <issue>, <fpage> e <lpage> não são inseridos no formato *xmlwos*

### ISSUE THREAD

**Issue by amandasramalho** [scieloorg/articles_meta#201]
### Descrição do problema
O registro do documento S0253-570X2018000100010, da coleção cubana contém os metadados suficientes para que sejam representados utilizando os campos <volume>, <issue>, <fpage> e <lpage> quando no formato xmlwos, entretanto os elementos não constam no XML produzido.

### Passos para reproduzir o problema
1. O artigo é: http://scielo.sld.cu/scielo.php?script=sci_arttext&pid=S0253-570X2018000100010&lng=es&nrm=iso&tlng=es
2. Pacote XML: scielo_articles.zip
3. \cub\0253-570X\S0253-570X2018000100010.xml

### Comportamento esperado
No arquivo XML disponível deveria conter os elementos: <volume>, <issue>, <fpage> e <lpage> do artigo.

### Origem do erro:
Foi reportado originalmente pela OCLC, que coleta estes dados para os serviços de descoberta.
Mensagem original:
Can you also ask them to confirm that all metadata elements/tags are being collected as well? For example, for same journal I referenced earlier (ISSN: 2224-4700) the following elements/tags and values are often missing from articles dated 2016 and later (see the attached example):
<volume>
<issue>
<fpage>
<lpage>
I’m not sure if this is true of other journals. Many of the ones I checked have consistently have these tags, but I only checked a small sample articles from different journals in the database. The absence of these tags is not required for ingest into the central index, but having this metadata in the record would improve knowledge base OpenURL links to your platform from these records.

**Comment by robertatakenaka** [comment]
@scieloorg/scielo-brazil-developers me parece que o problema é que o dicionário (JSON) não contém a chave "issue", no entanto, os dados da chave "article" contém os dados suficientes para gerar `raw.issue` e `raw.volume`. De fato, não existe, `raw.fpage` nem `raw.lpage`.

**Comment by robertatakenaka** [comment]
Entrei em contato com a coleção Cuba para que o dado seja corrigido

**Comment by robertatakenaka** [comment]
Estivemos em contato com Cuba e não entendemos como o script não funciona ao ser executado. No entanto, ao fazer manualmente os comandos de ftp. A transferência ocorre. 
Vamos seguir trabalhando em deixar automático.
Mas por enquanto a base está atualizada com a data de 25 de março de 2020. E possivelmente corrija o problema.

**Comment by amandasramalho** [comment]
Como o pacote scielo_articles.zip é atualizado semanalmente às terças-feiras, na próxima terça possivelmente os registros estarão corrigidos, é isso? Ou o pacote já foi reprocessado com a correção?

**Comment by gustavofonseca** [comment]
Sim, mas vai ser necessário remover os XMLs do disco do local onde o pacote `scielo_articles.zip` é gerado.

`\cub\0253-570X\S0253-570X2018000100010.xml` é o único que deverá ser corrigido?

**Comment by amandasramalho** [comment]
Acredito que tenham mais artigos com o mesmo problema, esse artigo foi somente para usar uma referência. Não analisei exaustivamente...

**Comment by gustavofonseca** [comment]
Todos restritos à coleção Cuba?

**Comment by amandasramalho** [comment]
Vou baixar o pacote e olhar, a OCLC me deu esse exemplo mas não me disse se acontecia em mais coleções.

**Comment by gustavofonseca** [comment]
Ok, aguardo tua resposta.

**Comment by amandasramalho** [comment]
@gustavofonseca, olhei todas as coleções por amostragem e aparentemente só acontece com Cuba mesmo.

**Comment by gustavofonseca** [comment]
Ok, vou forçar que todos os documentos de Cuba sejam atualizados no pacote que será disponibilizado na próxima terça-feira. Podemos fechar este ticket?

**Comment by amandasramalho** [comment]
Eu fecharia após validação. Na terça mesmo posso baixar os arquivos atualizados e dar uma olhada.

**Comment by gustavofonseca** [comment]
Nós podemos adiantar essa validação acessando `http://articlemeta.scielo.org/api/v1/article/?collection=cub&code=S0253-570X2018000100010&format=xmlwos`, e conferindo a presença dos dados (acabo de ver que não estão presentes). @robertatakenaka, é esperado que os dados não estejam presentes neste momento?

**Comment by robertatakenaka** [comment]
@gustavofonseca : só podemos garantir que os dados estão atualizados no ftp esperado, nada mais.

**Comment by gustavofonseca** [comment]
> @gustavofonseca : só podemos garantir que os dados estão atualizados no ftp esperado, nada mais.

Ok. Aguardaremos a execução dos processamentos diversos e verificamos na próxima semana. Grato!

**Comment by amandasramalho** [comment]
@gustavofonseca @robertatakenaka 
Baixei o pacote atualizado hoje e verifiquei os registros, continuam sem os dados de <volume>, <issue>, <fpage> e <lpage>. A data do arquivo continua sendo de 10/12/2019.
